.env

```diff
@@ -0,0 +1,11 @@
+## Firebase
```

Comment:
hmm we shouldn't have this file on repo

Reply:
but i added this one to gintignore ?

Reply:
you've added directory, not file 
Is `.env/`, should be `.env`

Reply:
hmmmm before i added only .env and was the same situation, in this moment i added .env